runtime/lua/vim/lsp/protocol.lua

```diff
@@ -898,7 +898,8 @@ function protocol.resolve_capabilities(server_capabilities)
   elseif type(textDocumentSync) == 'number' then
     -- Backwards compatibility
     if not TextDocumentSyncKind[textDocumentSync] then
-      return nil, 'Invalid server TextDocumentSyncKind for textDocumentSync'
+      vim.notify('Invalid server TextDocumentSyncKind for textDocumentSync', vim.log.levels.ERROR)
```

Comment:
was the returned message not being logged?

Reply:
idk. check this function usage . it not handle this error before so i am not sure it should be logged or not . just use notify through it